specs/validator/tableland-openapi-spec.yaml

```diff
@@ -119,9 +143,11 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/TxnReceipt"
+                $ref: "#/components/schemas/TransactionReceipt"
         "400":
-          description: Invalid query/statement value
+          description: Invalid chain identifier or transaction hash format.
+        "404":
+          description: There's no transaction receipt with the provided hash.
```

Comment:
I think this is what we have now as well, correct?

Reply:
Ah ok gotcha. Well either way, I like the 404 in that case. It is being explicit about the fact that the validator literally can't find that transaction. It is helpful from the client's perspective because now I can deal with the fact that the validator is unaware of a txn I am expecting to exist. So yes good idea!